UI.AutomationTests/Admin/Booking/EditMultiDayHearingTests.cs

```diff
@@ -0,0 +1,194 @@
+using UI.PageModels.Pages.Admin.Booking;
+
+namespace UI.AutomationTests.Admin.Booking
+{
+    public class EditMultiDayHearingTests : AdminWebUiTest
+    {
+        [Category("Daily")]
+        [Test]
+        public void EditSingleDayOfMultiDayHearing()
+        {
+            var isV2 = FeatureToggles.UseV2Api();
+            const int numberOfDays = 3;
+            var hearingDto = HearingTestData.CreateMultiDayDtoWithEndpoints(numberOfDays, DateTime.Today.AddDays(1).AddHours(10).AddMinutes(00));
+            var bookingDetailsPage = BookMultiDayHearingAndGoToDetailsPage(hearingDto);
+            UpdateCaseName(hearingDto, numberOfDays);
+            
+            // Change the scheduled datetime
+            var newTime = hearingDto.ScheduledDateTime.AddMinutes(30);
+            var summaryPage = bookingDetailsPage.UpdateSchedule(newTime, hearingDto.DurationHour, hearingDto.DurationMinute);
+            hearingDto.ScheduledDateTime = newTime;
+            
+            // Change everything else
+            summaryPage = EditMultiDayHearing(isV2, hearingDto, summaryPage);
+            
+            summaryPage.ValidateSummaryPage(hearingDto);
+            var confirmationPage = summaryPage.ClickBookButton();
+            
+            confirmationPage.IsBookingSuccessful().Should().BeTrue();
+            var bookingDetailPage = confirmationPage.ClickViewBookingLink();
+            bookingDetailPage.ValidateDetailsPage(hearingDto);
+            Assert.Pass();
+        }
+
+        [Category("Daily")]
+        [Test]
+        public async Task EditThisAndUpcomingDaysOfMultiDayHearing()
+        {
+            var isV2 = FeatureToggles.UseV2Api();
+            var multiDayBookingEnhancementsEnabled = FeatureToggles.MultiDayBookingEnhancementsEnabled();
+            if (!isV2 || !multiDayBookingEnhancementsEnabled)
+                Assert.Pass("V2 and multi day booking enhancements are not both enabled, cannot edit this and upcoming hearings. Skipping Test");
+            
+            const int numberOfDays = 3;
+            var hearingDto = HearingTestData.CreateMultiDayDtoWithEndpoints(numberOfDays, DateTime.Today.AddDays(1).AddHours(10).AddMinutes(00));
```

Comment:
Will this be safe to run over weekends and bank holidays?

Reply:
that's a good point, thanks. For simplicity I've changed the tests to schedule the hearings for Monday on the following week, that way there won't be any overlap with the weekend.

bank holidays shouldn't be a problem I don't think. I believe there used to be a restriction in place to avoid scheduling on those holidays, but it was removed